docs/content/3.manual/manual.yml

```diff
@@ -1739,6 +1739,10 @@ sections:
 
             The input is converted to base64 as specified by RFC 4648.
 
+          * `@base64d`:
+
+            The inverse of `@base64`, input is decoded as specified by RFC 4648.
+
```

Comment:
Please explain that if the decoded string is not UTF-8 then jq may or may not raise an error.